test/controllers/api/docs_controller_test.exs

```diff
@@ -8,6 +8,8 @@ defmodule HexWeb.API.DocsControllerTest do
   setup do
     User.create(%{username: "eric", email: "eric@mail.com", password: "eric"}, true)
     |> HexWeb.Repo.insert!
+    File.rm_rf!(Application.fetch_env!(:hex_web, :tmp_dir))
+    File.mkdir!(Application.fetch_env!(:hex_web, :tmp_dir))
```

Comment:
not super happy about this. This is needed to have reliable tests for a case when the pre-release is the very first release and we probably want to push it as top-level (see: https://github.com/wojtekmach/hex_web/commit/c3f6c6ea983c177d955c720df8f8b8cb527de4d1)